c/polyjuice_globals.h

```diff
@@ -1,18 +1,40 @@
 #ifndef POLYJUICE_GLOBALS_H
 #define POLYJUICE_GLOBALS_H
 
-#define POLYJUICE_VERSION "v0.8.10-aot"
-#define POLYJUICE_SHORT_ADDR_LEN 20
-/* 32 + 4 + 20 */
-#define SCRIPT_ARGS_LEN 56
+#define POLYJUICE_VERSION "v1.1.1"
+
+#define ETH_ADDRESS_LEN 20
+
+/* Key type for ETH Address Registry */
+#define GW_ACCOUNT_SCRIPT_HASH_TO_ETH_ADDR 200
+#define ETH_ADDR_TO_GW_ACCOUNT_SCRIPT_HASH 201
+
+/** Polyjuice contract account (normal/create2) script args size */
+#define CONTRACT_ACCOUNT_SCRIPT_ARGS_LEN 56 /* 32 + 4 + 20 */
+#define CREATOR_SCRIPT_ARGS_LEN 40			/* 32 + 4 + 4  */
```

Comment:
Should be 36.

32 bytes rollup scripts hash | 4 bytes sudt_id

the registry id is fixed to ETH_REGISTRY_ACCOUNT_ID.

Reply:
`gw-tools create_creator_account` created a creator_account with CREATOR_SCRIPT_ARGS_LEN = 40.

see:
https://github.com/nervosnetwork/godwoken/blob/982c879/crates/tools/src/create_creator_account.rs#L57-L59

Reply:
0d94b46d10dfc36f12e65154dfc72cb68aac28b7